controllers/club/settings/invite_members/impl/emailNewMembers.js

```diff
@@ -43,13 +47,27 @@ module.exports = async (req, res) => {
                         const settings_message = {User: user.firstName + " " + user.lastName, Type: `Invited user via email to join club`, Date: date, Time: time}
                         club.settings_history.unshift(settings_message)
                         club.save()
+                        
```

Comment:
Is this all tabbed like this? i would move it back some but not sure if git is making it weird

Reply:
you didn't expand the code it's spaced correctly